kuma/core/middleware.py

```diff
@@ -296,9 +301,24 @@ def __call__(self, request):
             pass
         else:
             # HTTP_X_FORWARDED_FOR can be a comma-separated list of IPs.
-            # The client's IP will be the first one.
-            forwarded_for = forwarded_for.split(",")[0].strip()
-            request.META["REMOTE_ADDR"] = forwarded_for
+            forwarded_for_ips = forwarded_for.replace(",", " ").split()
+            if (settings.PROXY_DEPTH > 0) and len(
+                forwarded_for_ips
+            ) >= settings.PROXY_DEPTH:
+                # The trusted client IP is selected relative to the end of the list,
+                # depending on settings.PROXY_DEPTH. For example, if we received:
+                #     ["1.1.1.1", "2.2.2.2", "3.3.3.3", "4.4.4.4"]
+                # and the PROXY_DEPTH was set to 2, the trusted client IP would be
+                # second from the end, or "3.3.3.3". This is because intermediate
+                # proxies (like our CDN and our ELB) append the actual client IP
+                # of their connection to the end of the header.
+                client_ip = forwarded_for_ips[-settings.PROXY_DEPTH]
+                try:
+                    request.META["REMOTE_ADDR"] = str(ip_address(client_ip))
```

Comment:
That means that in local dev you never get a `request.META["REMOTE_ADDR"]` set whereas we used to before. Does that matter?

Reply:
In local dev, I think something else sets `REMOTE_ADDR`, because in my local testing, the `X-Forwarded-For` header is never set (and shouldn't be), so as far as I can tell this code never sets it.